Swiften/Elements/MIXParticipant.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2017 Tarun Gupta
+ * Licensed under the simplified BSD license.
+ * See Documentation/Licenses/BSD-simplified.txt for more information.
+ */
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include <boost/optional.hpp>
+
+#include <Swiften/Base/API.h>
+#include <Swiften/Elements/Payload.h>
+#include <Swiften/JID/JID.h>
+
+namespace Swift {
+    class SWIFTEN_API MIXParticipant : public Payload {
+
+        public:
+            typedef std::shared_ptr<MIXParticipant> ref;
```

Comment:
@Kev Should we reflect this in https://github.com/swift/swift/wiki/Coding-Style ?